config/initializers/omniauth.rb

```diff
@@ -1,6 +1,15 @@
 Rails.application.config.middleware.use OmniAuth::Builder do
   ENV["GITHUB_KEY"] ||= "03fafb19e89f60f1f0f3"
   ENV["GITHUB_SECRET"] ||= "2fd993d61ff8eefa127ee5eee0507bdfa5312c44"
+
   provider :github, ENV["GITHUB_KEY"], ENV["GITHUB_SECRET"],
            scope: "user,repo"
 end
+
+unless Rails.env.production? || Rails.env.test?
+  OmniAuth::Strategies::OAuth2.class_eval do
```

Comment:
As a side note, I am wondering if we could use the forwarder even in production. My main concern is this could expose us to some weird security stuff, but I am by no means an expert on the subject. What do you think, @exalted?